src/scss/_projects.scss

```diff
@@ -12,5 +15,10 @@
         font-size: $p-font-size;
```

Comment:
Why do you repeat font-size here? Do you have a general font-size in `body` tag?

Reply:
will be later